modin/logging/config.py

```diff
@@ -85,7 +87,7 @@ def bytes_int_to_str(num_bytes, suffix="B"):
         if num_bytes < factor:
             return f"{num_bytes:.2f}{unit}{suffix}"
         num_bytes /= factor
-    return f"{num_bytes:.2f}{1000+P}{suffix}"
+    return f"{num_bytes * 1000:.2f}P{suffix}"
```

Comment:
Wait was this a typo in the original code? I can't tell what `P` is referring to?

Reply:
Not a typo, this case is hit when we have an Exabyte (1000+ Petabytes) of system memory which will likely never happen. In the original code, we just naively print out `num_bytes 1000+ PB`. This have been modified to actually multiply `num_bytes` by 1000 and then print out the number of PBs which I think is cleaner.

Reply:
Resolved - my confusion came from the bug in the original code.